consumer/kafka/src/main/java/com/spotify/heroic/consumer/kafka/ConsumerThread.java

```diff
@@ -129,58 +129,58 @@ private void threadIsStopping() {
         return async.resolved();
     }
 
-    public boolean isPaused() {
-        return this.paused.get() != null;
+    public boolean getShouldPause() {
```

Comment:
`is*` is convention for accessing booleans,  so `isPaused()` or `isPausing()` might be better names here..